### PR TITLE
Fix #3236: Hi-Fi Add Profile Tablet

### LINKED ISSUE #3236: Hi-Fi Add Profile Tablet
Mobile - https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/

Tablet - https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/271b428d-7d98-40b4-b74e-f2a5266b0587/

**Follow this guide**: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by viktoriias**
## Explanation

Fixes #3236: Hi-Fi Add Profile Tablet.
- Remove `layout-land`, `layout-sw600dp-port` and `layout-sw600dp-land` files
- Introduce dimensions resources
- Change mobile layout to be of fixed width (see [comment](https://github.com/oppia/oppia-android/pull/3710#pullrequestreview-736061200))

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by viktoriias** [comment]
**Tablet**

Before             |  After
:-------------------------:|:-------------------------:
  |

**Comment by viktoriias** [comment]
**Tablet**

Before             |  After
:-------------------------:|:-------------------------:
  |

**Comment by viktoriias** [comment]
**Phone**

Before             |  After
:-------------------------:|:-------------------------:
  |  
  |

**Comment by viktoriias** [comment]
**Phone**

Before             |  After
:-------------------------:|:-------------------------:
  |  
  |  
  |

**Comment by viktoriias** [comment]
@rt4914 PTAL if this PR satisfies the latest requirements, as mentioned in https://github.com/oppia/oppia-android/pull/3710#pullrequestreview-736061200

File: app/src/main/res/layout/add_profile_activity.xml [hunk 1]
```diff
@@ -250,4 +241,4 @@
         android:background="@drawable/toolbar_drop_shadow" />
     </FrameLayout>
   </LinearLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/add_profile_activity.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — APPROVED** [review]
@viktoriias One nit change suggested.

File: app/src/main/res/layout/add_profile_activity.xml (hunk 1 above)
Comment (viktoriias) [reply app/src/main/res/layout/add_profile_activity.xml]: Done.

**Comment by rt4914** [comment]
@viktoriias There are actually merge conflicts, can you please update this and merge directly?

**Comment by viktoriias** [comment]
> @viktoriias There are actually merge conflicts, can you please update this and merge directly?

Merging is done, but I'm "not authorized to merge this pull request".

**Pull request merged**